.pre-commit-config.yaml

```diff
@@ -1,12 +1,69 @@
+# Learn more about this config here: https://pre-commit.com/
+
+# To enable these pre-commit hooks run:
+# `brew install pre-commit` or `python3 -m pip install pre-commit`
+# Then in the project root directory run `pre-commit install`
+
 default_stages: [commit]
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
+    hooks:
+      - id: check-builtin-literals
+      - id: check-case-conflict
+      - id: check-docstring-first
+      # - id: check-executables-have-shebangs  # TODO: Enable and fix or delete
+      # - id: check-json  # TODO: Enable and fix or delete
```

Comment:
This one is already in #245 

I think it's appropriate to remove the TODO ones.

Reply:
I actually think TODOs are a great way to say "Someone should fix this (AWS keys are checked in) but I do not know how to do it".  I find that TODOs are better than sweeping the dust under the rug.